Add how to Post/Get/List orders (#2)
spacefill/rest-api-tutorial: crud/orders.md

# Orders CRUD

> See complete API orders endpoints in https://api.sandbox.spacefill.fr/docs

Reviewer: ```suggestion
See complete orders API endpoints on https://api.sandbox.spacefill.fr/docs
```

## Examples

> Each requests use [`jq`](https://stedolan.github.io/jq/) to parse and render JSON, don't forget to remove it if you do some copy/paste.

Reviewer: ```suggestion
In the following examples, each request uses [`jq`](https://stedolan.github.io/jq/) to parse and render the JSON, don't forget to remove it if you do some copy/paste.
```
J'ai rajouté le "In the following examples" parce que sinon ça me mettait le doute de si on recommandait d'utiliser jq ou non

### Create

> To create an entry `order` with those items:

Reviewer: Ici on n'a pas d'items défini avant donc je reformule un peu (sinon on se demande de quels items on parle)

```suggestion
To create an entry `order` with two items:
```

  "exit_final_recipient_address_line1": null,
  "exit_final_recipient_address_line2": null,
  "exit_final_recipient_address_line3": null,
  "exit_final_recipient_address_zip": null,
  "exit_final_recipient_address_city": null,
  "exit_final_recipient_address_country": null,
  "exit_final_recipient_planned_datetime_range": null,
  "created_at": "2021-09-29T13:20:59.286179+00:00",
  "created_by": "4b03020e-007b-4430-86fd-dc409f6a0001",
  "updated_at": "2021-09-29T13:20:59.286179+00:00",
  "updated_by": "4b03020e-007b-4430-86fd-dc409f6a0001",
  "order_items": [
    {
      "id": "107c0bb0-4173-4d86-9685-8f49c4800001",
      "master_item_id": "13acc10a-a6ab-4099-b600-fb33fa6c0001",
      "batch_id": "e4f77256-304a-4911-a1eb-9f6d47ca0001",
      "position": 1,
      "item_packaging_type": "PALLET",
      "expected_quantity": 1,
      "actual_quantity": null
    },
    {
      "id": "107c0bb0-4173-4d86-9685-8f49c4800002",
      "master_item_id": "13acc10a-a6ab-4099-b600-fb33fa6c0002",
      "batch_id": "e4f77256-304a-4911-a1eb-9f6d47ca0005",
      "position": 2,
      "item_packaging_type": "PALLET",
      "expected_quantity": 1,
      "actual_quantity": null
    }
  ]
}
```

> Or an exit `order` with those items:

Reviewer: Comme on est pas dans une phrase comme dans la doc générale, je reprendrais plutôt le style du titre des entrées:
```suggestion
To create an exit `order` with two items:
```

      "position": 1,
      "item_packaging_type": "PALLET",
      "expected_quantity": 5,
      "actual_quantity": 5
    },
    {
      "id": "107c0bb0-4173-4d86-9685-8f49c4800002",
      "master_item_id": "13acc10a-a6ab-4099-b600-fb33fa6c0002",
      "batch_id": "e4f77256-304a-4911-a1eb-9f6d47ca0005",
      "position": 2,
      "item_packaging_type": "CARDBOARD_BOX",
      "expected_quantity": 10,
      "actual_quantity": 9
    },
    {
      "id": "107c0bb0-4173-4d86-9685-8f49c4800003",
      "master_item_id": "13acc10a-a6ab-4099-b600-fb33fa6c0008",
      "batch_id": "e4f77256-304a-4911-a1eb-9f6d47ca0013",
      "position": 3,
      "item_packaging_type": "EACH",
      "expected_quantity": 20,
      "actual_quantity": 20
    },
    {
      "id": "107c0bb0-4173-4d86-9685-8f49c4800004",
      "master_item_id": "13acc10a-a6ab-4099-b600-fb33fa6c0009",
      "batch_id": null,
      "position": 4,
      "item_packaging_type": "PALLET",
      "expected_quantity": 30,
      "actual_quantity": 30
    }
  ]
}
```

## List

```sh
$  curl -sLX 'GET' \
  'https://api.sandbox.spacefill.fr/v1/logistic_management/orders' \
  -H 'accept: application/json' \

>   -H 'x_token: shipper_api_token_0001' | jq

Reviewer: @Its-Alex 

```suggestion
  -H 'x_token: secret' | jq
```

# Orders CRUD

> 

Reviewer 2: @Its-Alex **suggestion:** add here:

```
See complete API orders endpoints in https://api.sandbox.spacefill.fr/docs
```
